### PR TITLE
[Mobile Payments] Update URL to wp-admin

### LINKED ISSUE #5616: [Mobile Payments]: Wrong path for payments setup for some stores
While I was looking at some details in our codebase of WCiOS, I discovered that we hardcoded a path for payments setup here: https://git.io/JMD7l I believe that if the user has set a custom path for `wp-admin`, the payments setup would not work. It would be ideal, in my opinion, to substitute the initial wp-admin path with the new `adminURL` introduced by @itsmeichigo here https://github.com/woocommerce/woocommerce-ios/pull/5611

### PR DESCRIPTION
**Pull request by ctarda**
Closes: #5616 

### Description

Update the URL to wp-admin to rely on the store's adminURL.

### Testing instructions

1. Log in to a store with WooCommerce Payments activated but not setup completely. 
2. Navigate to Settings > In-Person Payments
3. Notice the error, and the url in the web view.

---
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

File: WooCommerce/WooCommerce.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -820,9 +820,6 @@
 		5783FB3F25D7369F00B9984B /* WooAnalyticsEventPropertyType.swift in Sources */ = {isa = PBXBuildFile; fileRef = 5783FB3E25D7369F00B9984B /* WooAnalyticsEventPropertyType.swift */; };
 		57896D6625362B0C000E8C4D /* TitleAndEditableValueTableViewCellViewModel.swift in Sources */ = {isa = PBXBuildFile; fileRef = 57896D6525362B0C000E8C4D /* TitleAndEditableValueTableViewCellViewModel.swift */; };
 		5791FB4224EC834300117FD6 /* MainTabViewModelTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = 5791FB4124EC834300117FD6 /* MainTabViewModelTests.swift */; };
-		5795F22C23E26A8D00F6707C /* OrderSearchStarterViewController.xib in Resources */ = {isa = PBXBuildFile; fileRef = 5795F22B23E26A8D00F6707C /* OrderSearchStarterViewController.xib */; };
```
Comment (allendav) [inline WooCommerce/WooCommerce.xcodeproj/project.pbxproj]: What happened here? I see many deletions in the project file besides for Site+Woo. I assume these were accidental?

File: WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Onboarding Errors/InPersonPaymentsWCPayNotSetup.swift [hunk 2]
```diff
@@ -37,12 +37,12 @@ struct InPersonPaymentsWCPayNotSetup: View {
     }
 
     var setupURL: URL? {
-        ServiceLocator.stores.sessionManager.defaultSite?.adminURL(path: Constants.wcpaySetupPath)
-    }
-}
+        guard let adminURL = ServiceLocator.stores.sessionManager.defaultSite?.adminURL else {
```
Comment (allendav) [inline WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Onboarding Errors/InPersonPaymentsWCPayNotSetup.swift:40]: I'm having trouble hitting this use case. I have a site with WCPay active but not setup, and it hits this

https://github.com/woocommerce/woocommerce-ios/blob/81ed7b18078c58d3337f53a91d0f09c802bf3c3f/WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person%20Payments/CardPresentPaymentsOnboardingUseCase.swift#L144

before it gets to the `isWCPaySetupCompleted` that would trigger this flow.

WCPay Dev shows this in the account cache as expected for a freshly activated plugin without an account set up:

```
array (
  'account' => 
  array (
  ),
  'expires' => 1673396118,
)
```

I'll keep trying, but these changes look good and other than the concern about the project file noted below, ship it :)

Comment (allendav) [reply WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Onboarding Errors/InPersonPaymentsWCPayNotSetup.swift:40]: I ended up completing setup on this site and then forcing isWCPaySetupCompleted to return false. I then got the webview with the correct url.

**Review by allendav — APPROVED** [review]
Approved, but please double check project file changes before committing - some of those don't look right.

File: WooCommerce/WooCommerce.xcodeproj/project.pbxproj (hunk 1 above)
Comment (ctarda) [reply WooCommerce/WooCommerce.xcodeproj/project.pbxproj]: I have no idea what happened here, but none of this classes are used in the code anywhere, and the files are not on disk. 

I am going to replicate the PR in a separate branch, and see what happens

Comment (ctarda) [reply WooCommerce/WooCommerce.xcodeproj/project.pbxproj]: Okay, if I branch out from trunk and remove Site+Woo.swift from the project, the same changes happen again. Which does not make any sense

**Comment by ctarda** [comment]
The out of scope changes to the project file where fixed in trunk, so I'll merge now

**Pull request merged**